### DISCUSSION TITLE
How Add Fields to New Chekout?

### DISCUSSION THREAD

**Discussion by axelruiz** [woocommerce/woocommerce#45406] [Checkout Flow]
So pretty the new block for checkout, but wow i add a simple field in these new block ?

**Comment by senadir** [comment]
Hey! If you're comfortable with code, you can follow [this tutorial to add extra fields in Checkout.](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/additional-checkout-fields.md)
